### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mariogamer2**

### CODE REVIEW

**Comment by mariogamer2** [comment]
Must be good now.

File: README.md [hunk 1]
```diff
@@ -67,7 +71,7 @@ sdmmc.c & sdmmc.h originally written by Normatt
 Licensed under GPLv2 or any later version, refer to the license.txt file included.
 
 * Smealum and contributors for libctru
-* Normatt for sdmmc.c and .h, and also for .ld files and the log from 3dmoo9 that provided us with some of the information needed to get screen init
+* Normatt for sdmmc.c and .h, and also for .ld files and the log from XDS by ichfly that provided us some of the information needed to get screen init
```
Comment (Normmatt) [inline README.md]: Normmatt has 2 m's

**Comment by mariogamer2** [comment]
@Normmatt Don't understand?

lol,ok understand XD

**Comment by Normmatt** [comment]
@mariogamer2 I mean my nick is spelled wrong "Normatt" should be "Norm**m**att"

**Comment by pixel-stuck** [comment]
you should squash all of those commits into one... 

git rebase -i HEAD~11

change all but one to fixup and then

git commit 
git push --force

**Comment by mariogamer2** [comment]
Ok @dark-samus I'll do thank.

**Comment by mariogamer2** [comment]
There is some problem....

I cannot push,its give me already up-to-date

**Comment by pixel-stuck** [comment]
@mariogamer2 that's why you do 

git push --force

**Comment by mariogamer2** [comment]
Always not working...

**Comment by urherenow** [comment]
Fastest way? Cancel the request, delete your fork, re-fork it, make all required changes at once (within github in your browser) and submit new pull request.

**Comment by mariogamer2** [comment]
@urherenow I was thinking thats to XD

**Pull request closed without merging**